public/src/main/java/com/revenuecat/purchases/CustomerInfo.kt

```diff
@@ -42,13 +43,15 @@ data class CustomerInfo constructor(
     val allExpirationDatesByProduct: Map<String, Date?>,
     val allPurchaseDatesByProduct: Map<String, Date?>,
     val requestDate: Date,
-    val jsonObject: JSONObject,
+    @Deprecated(
+        "Use rawData instead"
```

Comment:
This looks good! I would also add the "replaceWith" parameter of the Deprecated annotation so the IDE suggests replacing jsonObject with rawData automatically.

```
@Deprecated(
    message = "Use rawData instead"
    replaceWith = ReplaceWith("rawData")
)
```

Reply:
Oh right thanks.